generators/chipyard/src/main/scala/ChipTop.scala

```diff
@@ -31,6 +31,7 @@ class ChipTop(implicit p: Parameters) extends LazyModule with BindingScope
   // of ChipTop (ex: ClockGroup) do not receive clock or reset.
   // However. anonymous children of ChipTop should not need an implicit Clock or Reset
   // anyways, they probably need to be explicitly clocked.
-  lazy val module: LazyModuleImpLike = new LazyRawModuleImp(this) {  }
+  lazy val module: LazyModuleImpLike = new ChipTopLazyRawModuleImp(this)
```

Comment:
Why do you need this?

Reply:
Don't even remember making that change, sorry please ignore any changes to files other than those in the upf folder and also the Makefile.

Reply:
Are you able to divorce your changes from the Floorplan aspects work?

Reply:
Yup I’ve taken all of the UPF related code out of floorplans files and made a separate upf folder to house it all. Just stuck on where in the flow my code should be called after running make verilog.